python/lsst/ip/diffim/psfMatch.py

```diff
@@ -1037,7 +1037,7 @@ def _solve(self, kernelCellSet, basisList, returnOnExcept=False):
 
         except pexExcept.Exception as e:
             pexLog.Trace(self.log.getName()+"._solve", 1, "ERROR: Unable to calculate psf matching kernel")
-            pexLog.Trace(self.log.getName()+"._solve", 2, e.args[0].what())
+            pexLog.Trace(self.log.getName()+"._solve", 2, e.args[0])
```

Comment:
e does not appear to work with 11.0.rc2 pex_logging that I'm testing with:
(Pdb) pexLog.Trace(self.log.getName()+"._solve", 2, e)
*** NotImplementedError: Wrong number or type of arguments for overloaded function 'new_Trace'.
  Possible C/C++ prototypes are:
    lsst::pex::logging::Trace::Trace(std::string const &,int const,std::string const,...)
    lsst::pex::logging::Trace::Trace(std::string const &,int const,char const *)
    lsst::pex::logging::Trace::Trace(std::string const &,int const,boost::format const &